recipe/meta.yaml

```
@@ -1,32 +1,36 @@
 {% set name = "scikit-build-core" %}
-{% set version = "0.6.1" %}
+{% set version = "0.9.9" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit_build_core-{{ version }}.tar.gz
-  sha256: 392254a4ca7235c27a4be98cc24cd708f563171961ce37cff66120ebfda20b7a
+  sha256: 64880e00c03e196ed2feb66676175dbbafe5fad1081edd9583c30bf6170d2fc9
+  patches:                                           # [win]
+    - patches/0001-register-cmake-exe-for-win.patch  # [win]
 
 build:
-  number: 1
+  number: 0
   skip: true  # [py<37]
 
 requirements:
+  build:
+    - m2-patch  # [win]
   host:
     - hatch-vcs
     - hatchling
     - pip
     - python
   run:
     - python
-    - exceptiongroup             # [py<311]
-    - importlib-metadata         # [py<38]
-    - importlib_resources >=1.3  # [py<39]
-    - packaging >=20.9
-    - tomli >=1.1                # [py<311]
-    - typing-extensions >=3.10   # [py<38]
+    - exceptiongroup >=1.0              # [py<311]
+    - importlib-metadata >=4.13         # [py<38]
+    - importlib_resources >=1.3         # [py<39]
+    - packaging >=21.3
+    - tomli >=1.2.2                     # [py<311]
+    - typing-extensions >=3.10          # [py<38]
```

```suggestion
    - typing-extensions >=3.10.0        # [py<39]
```
See https://github.com/scikit-build/scikit-build-core/blob/v0.9.9/pyproject.toml#L43